src/Features/LanguageServer/ProtocolUnitTests/Workspaces/LspWorkspaceManagerTests.cs

```diff
@@ -29,20 +29,20 @@ public async Task TestUsesLspTextOnOpenCloseAsync()
 
         await testLspServer.OpenDocumentAsync(documentUri, "LSP text");
 
-        var lspDocument = await GetLspDocumentAsync(documentUri, testLspServer).ConfigureAwait(false);
+        var (_, lspDocument) = await GetLspDocumentAsync(documentUri, testLspServer).ConfigureAwait(false);
```

Comment:
Can the testLspServer give some other way to get the workspace? This seems a bit icky if every time we're having to discard this, never mind that "GetLspDocumentAsync" isn't returning just the document anymore.

Reply:
as helper test code, this doesn't bother me :)